src/main/resources/assets/appliedenergistics2/lang/de_DE.lang

```diff
@@ -2,20 +2,20 @@
 tile.appliedenergistics2.cable_bus.name=AE2-Kabel und/oder Bus
 tile.appliedenergistics2.cell_workbench.name=Speicherzellenwerkbank
 tile.appliedenergistics2.charger.name=Ladegerät
-tile.appliedenergistics2.chest.name=ME-Truhe
+tile.appliedenergistics2.chest.name=ME Truhe
 tile.appliedenergistics2.condenser.name=Materiekondensator
-tile.appliedenergistics2.controller.name=ME-Controller
+tile.appliedenergistics2.controller.name=ME Controller
 tile.appliedenergistics2.crank.name=Holzkurbel
 tile.appliedenergistics2.creative_energy_cell.name=Kreative Energiezelle
-tile.appliedenergistics2.dense_energy_cell.name=Dichte Energiezelle
+tile.appliedenergistics2.dense_energy_cell.name=Komprimierte Energiezelle
```

Comment:
True, but the 1:1 translation makes no sense here in my opinion cause "Dicht" doesnt  really fit for that, thats why i looked for something better